schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,159 @@
+                                        ]
+                                    },
+                                    "properties": {
+                                        "type": "object",
+                                        "properties": {
+                                            "endpointStatus": {
+                                                "enum": [
+                                                    "Enabled",
+                                                    "Disabled"
+                                                ]
+                                            },
+                                            "targetResourceId": {
+                                                "type": "string"
+                                            },
+                                            "target": {
+                                                "type": "string"
+                                            },
+                                            "weight": {
+                                                "type": "integer",
+                                                "minimum": 1,
+                                                "maximum": 1000
+                                               },
+                                            "priority": {
+                                                "type": "integer",
+                                                "minimum": 1,
+                                                "maximum": 1000
+                                            },
+                                            "endpointLocation": {
+                                                "type": "string"
+                                            },
+                                            "minChildEndpoints": {
+                                                "type": "integer",
+                                                "minimum": 1
+                                            }
+                                        }
+                                    },
+                                    "required": [
+                                        "name",
+                                        "type",
+                                        "properties"
+                                    ]
+                                }
+                            },
+                            "additionalItems": true
+                        }
+                    },
+                    "required": [
+                        "trafficRoutingMethod",
+                        "dnsConfig",
+                        "monitorConfig"
+                    ],
+                    "additionalProperties": true
+                }
+            },
+            "required": [
+                "apiVersion",
+                "name",
+                "type",
+                "properties"
+            ],
+            "additionalProperties": true
```

Comment:
We have read-only properties that the user might carry-over when copy-pasting from a GET request (which is, I imagine how most templates get written) and which the service will ignore. Is your recommendation to be strict and not allow these?